powershell-gallery/docs-conceptual/powershellget/supported-repositories.md

```diff
@@ -235,12 +235,15 @@ package provided by the .NET Foundation. You can use this package to create an A
 that hosts a package feed on any Windows server running IIS. This server uses the NuGet v2 protocol.
 For more information, see [Working with local PSRepositories][02].
 
-The **Microsoft.PowerShell.PSResourceGet** cmdlets don't currently support this type of repository.
-
-<!-- The feed URI has the following format:
+The **Microsoft.PowerShell.PSResourceGet** module supports NuGet.Server feeds. The feed URI has the
+following format:
 
 `http://<server-host-name>/nuget`
 
+> [!NOTE]
+> The **Microsoft.PowerShell.PSResourceGet** module expects the URI to end with `/nuget`. Any URI
+> that ends with `/nuget` is treated as a NuGet.Server feed.
```

Comment:
If the feed _doesn't_ end with `/nuget`, does the command error or otherwise reject the URI?

Reply:
There are several tests of the URI to determine the feed type. See https://github.com/PowerShell/PSResourceGet/blob/c4c9b372e84b03b11a1d68ef54795000a6fd3ac0/src/code/RepositorySettings.cs#L821-L847

It must match one of the endings or results in `Unknown` case. The `/nuget` case is specific to this server type.